Readme.md

```diff
@@ -11,15 +11,21 @@ Simple flavour of node/iojs binary management, no subshells, no profile setup, n
 
 ## Installation
 
+With node/iojs already installed:
+
     $ npm install -g n
 
-or
+or, by cloning this repo and running:
 
-    $ make install
+    $ PREFIX=$HOME make install
 
-to `$HOME`. Prefix later calls to `n` with `N_PREFIX=$HOME`
+to install to `$HOME`. Prefix later calls to `n` with `N_PREFIX=$HOME` or export `N_PREFIX` in your shell initialization file.
```

Comment:
As for using just `make install`: I'm happy to elaborate; I stuck with `$HOME`, because that's what the existing text seemed to describe, albeit inconsistently.

However, note that just `make install` will try to install to `/usr/local`, which will typically require `sudo`.

I think it actually makes sense to suggest a _user-level_ install - either by demonstrating use of `[N_]PREFIX`, as I've suggested, and/or even changing the _default_ location to `$HOME` (which is part of what the original description seemed to imply).